P5/Source/Guidelines/en/CMC-ComputerMediatedCommunication.xml

```diff
@@ -0,0 +1,1330 @@
+      to an ongoing interaction are produced to perform a move as part of the further development of
+      the interactional sequence, for instance in chats or forum discussions. Contributions to joint
+      CMC activities may not all be directly interactional; some may be part of a collaborative
+      project of the involved individuals. For example, editing activities in a shared text editor
+      or whiteboard in parallel with an ongoing CMC interaction (chat, audio conversation, or
+      audiovideo conference) in the same CMC environment in which these editing activities are
+      discussed by the collaborators.</p>
+    <p>Basic units of CMC can be described according to three criteria: (i) the temporal properties
+      of how these contributions are produced by their creators, transmitted via CMC systems, and
+      made accessible for the recipients; (ii) whether the unit as a whole is realised in a verbal
+      or nonverbal mode; (iii) for verbal units: whether the unit is realised in the written or
+      spoken mode. A taxonomy of basic CMC units resulting from these criteria is given in
+      <ptr target="#cmcunits-taxonomy"/>.
+    </p>
+    <figure xml:id="cmcunits-taxonomy">
+      <graphic url="Images/cmcunits-taxonomy.png"/>
+      <head>Taxonomy of basic CMC units according to <ptr type="cit" target="#BIB_CMC_Core"/></head>
+    </figure>
+    <p>The most important distinction in <ref
+    target="#cmcunits-taxonomy">the CMC taxonomy</ref>
+      concerns the temporal nature of units exchanged via CMC technologies. The left part of the
+      taxonomy describes units that are performed (by a producer) and perceived (by a recipient) as
+      a continuous stream of behaviour. Units of this type can be performed as</p>
+    <list type="gloss">
+      <label rend="bold">spoken utterances,</label>
+      <item>i.e. stretches of speech which are produced to perform a speaker turn in a
+        conversation,</item>
+      <label rend="bold">bodily activity,</label>
+      <item> i.e. nonverbal behavior (gesture, gaze) produced to perform a speaker turn, either
+        performed by the real body of an interlocutor (e.g. in a video conference) or through the
+        virtual avatar of an interlocutor in a 3D environment,</item>
+      <label rend="bold">onscreen activities,</label>
+      <item> i.e. non-bodily forms of behaviour that are transmitted to the group of interacting or
+        coworking participants, for instance the editing of content in a shared text editor which
+        can be perceived by the other parties simultaneously (as may be the case in learning or
+        collaboration environments).</item>
+    </list>
+    <p>The right part of <ref target="#cmcunits-taxonomy">the taxonomy</ref> describes units in
+      which the production, transmission, and perception of contributions to CMC interactions are
+      organised in a strictly consecutive order: The content — verbal, nonverbal, or multimodal — of
```

Comment:
```suggestion
      organised in a strictly consecutive order: The content—verbal, nonverbal, or multimodal—of
```

Reply:
Applied in https://github.com/TEIC/TEI/pull/2537/commits/7ca18f41bc4f8544b249cfff84478c1003bdb634